### ISSUE TITLE
Custom version of QtFeedback

### ISSUE THREAD

**Issue by travier** [flathub/org.kde.kalk#4]
This application appears to be using a custom version of QtFeedback: https://github.com/flathub/org.kde.kalk/blob/master/org.kde.kalk.json#L37

Is there a particular reason for that? If features are missing, we should use patches on top of the official release tarballs instead.

**Comment by tsdgeos** [comment]
There's no KFeedback framework, this is qtfeedback, but yes, it's weird it's using a fork anyway

**Comment by tsdgeos** [comment]
yeah different types of feedback, the kde thing is about asking the user for things, the qt thing is about vibrating your phone :D

**Comment by espidev** [comment]
Hmm, would it be fine to just use the release version of qtfeedback @JBBgameich?

**Comment by JBBgameich** [comment]
The released version is totally fine, the only issue (and probably reason why the fork was used here) is that qmake makes it a lot harder to install to the `/app` prefix if the rest of Qt was installed to a different prefix.

**Comment by Justinzobel** [comment]
Looks like this was removed, closing the issue.
